components/TooltipWrapper.jsx

```diff
@@ -6,11 +6,13 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { throttle } from "lodash";
+import cx from 'classnames';
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 require('../styles/components/_tooltipWrapper.scss');
 
 const propTypes = {
+    className: PropTypes.string,
```

Comment:
probably should allow for multiple class names with an object and then ...object in the className property but we can iterate.